apps/clock/index.html

```diff
@@ -74,7 +75,5 @@
       <li role="presentation"><a id="timer-tab" aria-controls="timer-panel" aria-selected="false" role="tab" href="#timer-panel" data-l10n-id="timer">Timer</a></li>
       <li role="presentation"><a id="stopwatch-tab" aria-controls="stopwatch-panel" aria-selected="false" role="tab" href="#stopwatch-panel" data-l10n-id="stopwatch">Stopwatch</a></li>
     </ul>
```

Comment:
Yeah, that makes sense, I've removed them. I initially thought that was the original problem -- but it turns out, removing them didn't fix the original problem either -- l10n was instead loading "en-US" a full second before updating to the other locale. Either way, I agree that it's better to not have the english text there.